.vscode/settings.json

```diff
@@ -1,12 +1,17 @@
 {
```

Comment:
Please remove this file from git and gitignore it

Reply:
@XavierChanth  Should I also remove .vscode/launch.json?